src/ui/inspector/Inspector.vue

```diff
@@ -199,8 +199,8 @@
         },
         methods: {
             refreshComposition(selection) {
-                if (selection[0]) {
-                    let parentObject = selection[0].context.item;
+                if (selection && selection[0]) {
```

Comment:
Do we need to check if `selection` is defined?

Reply:
Good catch, selection is always defined as an array. I removed that check.

Reply:
Actually, we need to check for selection and selection[0] length be greater than 0.